infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/event/EventHandler.java

```diff
@@ -11,12 +11,12 @@
 package org.eclipse.che.workspace.infrastructure.kubernetes.namespace.event;
 
 /**
- * Defines the handling mechanism for Kubernetes container events.
+ * Defines the handling mechanism for Kubernetes events.
  *
  * @author Sergii Leshchenko
  */
-public interface ContainerEventHandler {
+public interface EventHandler {
```

Comment:
Would it more specified if we rename `EventHandler` to `PodEventHandler` since it handles only pods events?